dist/api_data.py

```diff
@@ -120,6 +120,14 @@ def __cmp__(self, other):
 
 # Per-file configuration
 file_config = format_meta + [
+    Config('access_pattern', 'default', r'''
+        It is recommended that workloads that consist primarily of
+        updates and/or point queries specify \c random.  Workloads that
+        do many cursor scans through large ranges of data specify
+        \c sequential and other workloads specify \c default.  The
+        option leads to an advisory call to an appropriate operating
+        system API where available''',
+        choices=['default', 'random', 'sequential']),
```

Comment:
I don't love ``default`` here - it's not clear what that means. Maybe ``mixed`` would fit better? or ``normal`` to match the flag that is being passed to ``fadvise`` (though I'm also inclined to skip the call completely in mixed mode).